code/modules/clothing/modular_armor/modular.dm

```diff
@@ -789,4 +790,143 @@
 /obj/item/clothing/head/modular/marine/m10x/leader
 	name = "\improper M11X pattern leader helmet"
 	desc = "A slightly fancier helmet for marine leaders. This one has cushioning to project your fragile brain."
-	soft_armor = list("melee" = 75, "bullet" = 65, "laser" = 50, "energy" = 50, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 50)
+	soft_armor = list("melee" = 75, "bullet" = 65, "laser" = 55, "energy" = 55, "bomb" = 50, "bio" = 50, "rad" = 50, "fire" = 50, "acid" = 50)
+
+//SOM modular armour
+
+/obj/item/clothing/suit/modular/som
+	name = "\improper SOM light battle armor"
+	desc = "The M-21 battle armor is typically used by SOM light infantry, or other specialists that require more mobility at the cost of some protection. Provides good protection without minor impairment to the users mobility. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```

Comment:
```suggestion
	desc = "The M-21 battle armor is typically used by SOM light infantry, or other specialists that require more protection at the cost of some mobility. A standard combat battle armor for regular SOM tropos. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```

Reply:
Nah, the heavy armour is the "standard", to compare to. SOM lads are build to fight marines. I still have a gorillion typos and stuff though